### ISSUE TITLE
Add .editorconfig for consistancy between contributors

### ISSUE THREAD

**Issue by stautonico** [stonith404/pingvin-share#47]
I was working on issue #28 and I realized that my editor kept auto-formatting the code in a way that was different from the rest of the code. I think it would be a good idea to add a `.editorconfig` file so that contributors' IDEs and formatting stay consistent.

editorconfig reference: https://editorconfig.org/

**Comment by stonith404** [comment]
Hmm okay that's strange because Prettier should already set a consistant formatting style. Which IDE do you use?

**Comment by stautonico** [comment]
I'm using WebStorm. I'm not seeing a `.prettierrc` file in the repo. That's the file that most IDEs use. Did you commit it?

**Comment by stonith404** [comment]
Yep that's true. Editorconfig isn't built in in VsCode, a plugin is required, so I don't really see a difference to prettier. Would you say editorconfig is better?

**Comment by stautonico** [comment]
Either one, they both do the same job. I personally like editorconfig, simpler to configure IMO, but I'd be happy with either one as most IDEs support both

**Comment by stonith404** [comment]
Personally I would recommend Prettier, because it integrates very well with ESLint. But does Prettier work for you in WebStorm? And is it formatted the same link on the repo?

**Comment by stautonico** [comment]
Yeah, prettier is also supported in most major IDEs. Running the `npm run format` works fine, but the auto-format done by my IDE doesn't follow that format. To make that work in most IDEs, you need to have a centralized prettier config or editorconfig file somewhere in the project.

**Comment by stonith404** [comment]
Hm okay that isn't ideal. Then let's use editorconfig, do you have already an editorconfig file as a template?

**Comment by stonith404** [comment]
I close this issue because Prettier should do the job.
